packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -28,6 +28,7 @@ export default function createConnector(connectorDesc) {
 
   return Composed => class Connector extends Component {
     static displayName = `${connectorDesc.displayName}(${getDisplayName(Composed)})`;
+    static defaultTheme = Composed.defaultTheme;
```

Comment:
@mthuret You may be interested in this change (and the next one in themeable.js). This is what I needed to be able to do:

```js
var theme = {
  ...RangeRatings.defaultTheme,
  class: 'override'
};
```

Since you already explored the CSS theming of react-instantsearch, do you think that's an OK way to expose this defaultTheme?